packages/flutter_tools/lib/src/windows/visual_studio.dart

```diff
@@ -147,20 +147,24 @@ class VisualStudio {
     return '2019';
   }
 
-  /// The path to vcvars64.bat, or null if no Visual Studio installation has
+  /// The path to CMake, or null if no Visual Studio installation has
   /// the components necessary to build.
-  String get vcvarsPath {
+  String get cmakePath {
     final Map<String, dynamic> details = _usableVisualStudioDetails;
     if (details.isEmpty) {
       return null;
     }
-    return _fileSystem.path.join(
+    return _fileSystem.path.joinAll(<String>[
```

Comment:
Is this covered by our existing doctor checks?

Reply:
The change below (line 220) makes it part of the `doctor` flow.